File: benchmarks/base64-encode/Main.java
import java.util.Base64;

public class Main {

    public static void main(String[] args) {
        String i1 = "Hello, World!";
        String i2 = "The quick brown fox jumps over the lazy dog.";
        String i3 = "AaBbCcDdEeFf123456_!@#&*^()$=+-";

        long before = System.nanoTime();

        Base64.getEncoder().encodeToString(i1.getBytes());
        Base64.getEncoder().encodeToString(i2.getBytes());
        Base64.getEncoder().encodeToString(i3.getBytes());

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}

File: benchmarks/base64-encode/main.go
package main

import "fmt"
import "time"
import "encoding/base64"

func main() {
	var i1 = "Hello, World!"
	var i2 = "The quick brown fox jumps over the lazy dog."
	var i3 = "AaBbCcDdEeFf123456_!@#&*^()$=+-"

	var before = time.Now().UnixNano()

	base64.StdEncoding.EncodeToString([]byte(i1))
	base64.StdEncoding.EncodeToString([]byte(i2))
	base64.StdEncoding.EncodeToString([]byte(i3))

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}
